.agents/commands/request-audit.md

```
@@ -0,0 +1,478 @@
+
+### Thread reply
+Reply to the parent message with this (click the reply icon on the parent):
+
+---
+{thread reply with display name mentions}
+---
+```
+
+3. Tell the user the file path so they can open it:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)